raster-foundry/raster-foundry
# Add baseline Flyway migrations

## Overview
Add baseline migrations for flyway

### Checklist

- ~Description of PR is in an appropriate section of the [changelog](https://github.com/raster-foundry/raster-foundry/blob/develop/CHANGELOG.md) and grouped with similar changes if possible~ _*No functional changes applied yet*_
- ~Styleguide updated, if necessary~
- ~Swagger specification updated~
- ~Symlinks from new migrations prese~nt or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions
### Test base-lining process
- Make sure you don't have any env specific migrations.
- run `./scripts/load_development_data`
- run forklift migrations in sbt using `mg migrate`
- run `./scripts/migrate baseline 1`
- run `./scripts/migrate migrate` and verify that the migrations run successfully
- Verify that the `__migrations__` table doesn't exist anymore
- Make a change to one of the repeating migrations (adding a new row to it or something) and verify that it runs again when you run `./scripts/migrate migrate`
- Navigate around the app: create projects, view orgs etc to make sure things are in a good state.


Closes #4911
